<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AnimSender</class>
    <widget class="QWidget" name="AnimSender">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>animsource</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <widget class="QGroupBox" name="groupBox_2">
                    <property name="title">
                        <string>Color settings</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                            <widget class="QColorDialog" name="colorDialog" native="true"/>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>Settings</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QPushButton" name="primaryColor">
                                <property name="autoFillBackground">
                                    <bool>true</bool>
                                </property>
                                <property name="text">
                                    <string>Primary color</string>
                                </property>
                                <property name="flat">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="secondaryColor">
                                <property name="autoFillBackground">
                                    <bool>true</bool>
                                </property>
                                <property name="text">
                                    <string>Secondary color</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                <item>
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string>Animation Type:</string>
                                        </property>
                                        <property name="buddy">
                                            <cstring>animationType</cstring>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QComboBox" name="animationType"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_3">
                                <item>
                                    <widget class="QLabel" name="label_2">
                                        <property name="text">
                                            <string>Frame rate:</string>
                                        </property>
                                        <property name="buddy">
                                            <cstring>animationSpeed</cstring>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QSpinBox" name="animationSpeed">
                                        <property name="suffix">
                                            <string>ms</string>
                                        </property>
                                        <property name="minimum">
                                            <number>16</number>
                                        </property>
                                        <property name="maximum">
                                            <number>3000</number>
                                        </property>
                                        <property name="singleStep">
                                            <number>10</number>
                                        </property>
                                        <property name="value">
                                            <number>500</number>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QColorDialog</class>
            <extends>QWidget</extends>
            <header location="global">qcolordialog.h</header>
            <container>1</container>
            <slots>
                <signal>currentColorChanged()</signal>
                <signal>currentColorChanged(const QColor)</signal>
            </slots>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
